mu4/domain/notation/notationdomainmodule.cpp

```diff
@@ -47,6 +46,8 @@ void NotationDomainModule::registerExports()
     std::shared_ptr<INotationReadersRegister> readers = std::make_shared<NotationReadersRegister>();
     readers->reg({ "mscz", "mscx" }, std::make_shared<MsczNotationReader>());
     framework::ioc()->registerExport<INotationReadersRegister>(moduleName(), readers);
+
+    Notation::init();
```

Comment:
Why did you move that?

Reply:
In this method, `MScore` is initialized, and the initialization of other subsystems depends on it. In this case, `MScore` initialization is more like export